source/getting-started/blackboard.md

```diff
@@ -0,0 +1,152 @@
+# Blackboard
+
+Now that our robot Larry can reliably drive from `A` to `B`, we want to tweak
+the hardware and increase the update rate of our cyclic sensor data. For
+example, sending out the ultrasonic sensor distance only every 100 ms might no
+longer be sufficient. Similarly, a user might want to configure the battery
+threshold at which Larry raises a low-battery alarm.
+
+As systems grow more complex, they often accumulate thousands of individual
+configuration settings. Some are shared between processes, others are
+process-specific. With hundreds of processes in play, managing configuration
+efficiently becomes critical.
+
+A naïve approach would be to use publish-subscribe: one process manages the
+configuration and publishes updates to all subscribers. But this can quickly
+turn into a memory bottleneck. Consider a 1 MB configuration shared with 1000
+processes. Since iceoryx2 always pre-allocates for the worst case, it must
+reserve 1001 MB of memory - one copy for the publisher and one potential copy
+for each subscriber. Apart from the memory waste, you also risk configuration
+inconsistencies.
+
+The solution is the blackboard messaging pattern: a key-value repository in
+shared memory. Each process can access exactly the entries it needs, such as the
+ultrasonic sensor’s update rate. Any shared-memory-compatible type can be stored
+in the blackboard.
+
+Let’s implement an example where a user app can configure the sensor update
+rate. A higher rate means Larry reacts to obstacles faster and can drive faster.
+
+## Writer
+
+As always, we start by creating a node:
+
+```rust
+use iceoryx2::prelude::*;
+
+let node = NodeBuilder::new().create::<ipc::Service>()?;
+```
+
+With the blackboard pattern, all key-value pairs must be defined when the
+service is created. Unlike publish-subscribe, you cannot announce a key without
+providing a value. That means exactly one process is responsible for creating
```

Comment:
```suggestion
providing a value. That means exactly one `participant` is responsible for creating
```
@elfenpiff 

In the documentation, the "process" is defined as the operating system construct, and a `participant` is defined as a thread of execution participating in communication.

We often conflate these concepts, but I think the distinction is necessary for precision...

i.e. Sometimes there are multiple participants in a single process, and some platforms do not have the notion of a process at all.

Could you revise the places where you use process and decide if you really mean a process, or if you had a participant in mind instead.